src/objective-c/ProtoCompiler-gRPCPlugin.podspec

```diff
@@ -41,7 +41,7 @@ Pod::Spec.new do |s|
   # Because CocoaPods installs pods in alphabetical order, beginning this pod's name with an
   # exclamation mark ensures that other "regular" pods will be able to find it as it'll be installed
   # before them.
-  s.name     = '!ProtoCompiler-gRPCPlugin'
+  s.name     = 'ProtoCompiler-gRPCPlugin'
```

Comment:
You need to check if the restriction documented here for the name still applies.

Reply:
@jcanizales not a contributor, but I'm curios what exactly do you mean here? What is `!` in name for?